### DISCUSSION TITLE
【委托】求助角色抽取建议

### DISCUSSION THREAD

**Discussion by SKWTEA** [share121/inter-knot#3316] [其他]
本人绳网小萌新一枚，现在有莱卡恩、苍角、艾莲，加密母带120抽，在考虑抽简•杜或者卡吕东之子的那个火异常（忘了叫啥了），望绳网的大佬们给点建议
![微信图片_20240819122745](https://github.com/user-attachments/assets/a1f0cf29-47f5-4dcc-8849-16aec3a5bdac)

**Comment by Loxic123** [comment]
喜欢哪个抽哪个<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Nie7bai** [comment]
楼上正解<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8701a3b5fc90a6c37ba5491913b8218f_2209157026344411896.png" alt="布洛妮娅 开心鸭" width="75" >

**Comment by SKWTEA** [comment]
大佬们你们这些表情是怎么发的啊

**Reply by Nie7bai** [reply]
[用户脚本]使用米游社表情包
搜索这个标题，看回复人数最多的那一篇帖子，里面有操作方法

**Reply by Icather** [reply]
> [用户脚本]使用米游社表情包 搜索这个标题，看回复人数最多的那一篇帖子，里面有操作方法

感谢指路
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >

**Comment by chenx198** [comment]
强度党看测评，xp党看老二

**Comment by Charles070** [comment]
色😍

**Comment by Aaron377-maeke** [comment]
简还是有强度，看能不能搞个格莉丝，毕竟简是异常队的
